src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12352,4 +12372,32 @@ public override void CreateAnalyzerWithinCompilation(CompilationStartAnalysisCon
                 );
         }
     }
+
+    [Generator]
+    internal class SimpleGenerator : ISourceGenerator
+    {
+        private readonly string _sourceToAdd;
+
+        public SimpleGenerator()
+        {
+            _sourceToAdd = string.Empty;
+        }
```

Comment:
Minor: Consider removing if not used.

Reply:
It's used implicitly by other tests. There are some that create anything they can find via reflection, so they need to have a parameter less ctor. I'll add a comment though to explain it.